#!/usr/bin/env python
PACKAGE = "racecar_obs_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("K_vx",  double_t,    0, "Proportional Gain for longitudinal velocity ", 10, 0, 100)
gen.add("K_vy", double_t, 0, "Proportional Gain for lateral velocity",    10, 0,   100)
gen.add("K_y", double_t, 0, "Proportional Gain for lateral position",    10, 0,   100)
gen.add("dx", double_t, 0, "Disturbance for longitudinal acceleration ",    0.5, 0,   10)
gen.add("dy", double_t, 0, "Disturbance for lateral acceleration",    0.5, 0,   10)
gen.add("allow_lane_change", bool_t, 0, "Allow dynamic obstacles to do lane change",    False)

exit(gen.generate("racecar_obs_detection", "racecar_obs_detection", "config"))